tests/bluetooth/audio/cap_commander/src/test_broadcast_reception.c

```diff
@@ -371,3 +411,220 @@ ZTEST_F(cap_commander_test_broadcast_reception, test_commander_reception_start_i
 	zexpect_call_count("bt_cap_commander_cb.broadcast_reception_start", 0,
 			   mock_cap_commander_broadcast_reception_start_cb_fake.call_count);
 }
+
+ZTEST_F(cap_commander_test_broadcast_reception, test_commander_reception_stop_default_subgroups)
+{
+	int err;
+
+	err = bt_cap_commander_register_cb(&mock_cap_commander_cb);
+	zassert_equal(0, err, "Unexpected return value %d", err);
+
+	err = bt_cap_commander_broadcast_reception_start(&fixture->start_param);
+	zassert_equal(0, err, "Unexpected return value %d", err);
+
+	zexpect_call_count("bt_cap_commander_cb.broadcast_reception_start", 1,
+			   mock_cap_commander_broadcast_reception_start_cb_fake.call_count);
+	zassert_equal_ptr(NULL,
+			  mock_cap_commander_broadcast_reception_start_cb_fake.arg0_history[0]);
```

Comment:
Shouldn't this never be NULL?

Reply:
in 'cap_commander_complete', where the callback functions are called, the parameter 0 is 'failed_conn', which is NULL in case there was no error